### PR TITLE
Np 48265 Expansion - Update scientific index endpoint

### PR DESCRIPTION
**Pull request by monaullah**

### CODE REVIEW

File: expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java [hunk 1]
```diff
@@ -670,7 +670,7 @@ void shouldExpandPublicationWithNviStatusWhenPublicationIsReportedNviCandidate()
         var nviStatusNode = json.get(ScientificIndex.SCIENTIFIC_INDEX_FIELD);
 
         assertThat(nviStatusNode.get("year").asText(), is(equalTo("2024")));
-        assertThat(nviStatusNode.get("status").asText(), is(equalTo("Reported")));
+        assertThat(nviStatusNode.get("status").asText(), is(equalTo("REPORTED")));
```
Comment (monaullah) [inline expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java:673]: @truhacevkir Might this be a problem for `search-api`?

Comment (truhacevkir) [reply expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java:673]: I dont think so, I mean search does simple aggregation for scientific index statuses, frontend on other side should be aware of this change.

**Review by truhacevkir — APPROVED** [review]
Looks good, but look at my comment, important to fix this.

File: expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java (hunk 1 above)
Comment (monaullah) [reply expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java]: Changed json serialization, so it should be serialized "Reported" now

**Pull request merged**